README.md

```diff
@@ -2,4 +2,20 @@
 
 [![Build Status](https://travis-ci.org/qiime2/qiime2.svg?branch=master)](https://travis-ci.org/qiime2/qiime2)
 
-Source code repository for the QIIME 2 framework. Visit [https://qiime2.org](https://qiime2.org) to learn more about the QIIME 2 project.
+Source code repository for the QIIME 2 framework. 
+
+QIIME 2™ is a powerful, extensible, and decentralized microbiome bioinformatics platform that is free, open source, and community developed. With a focus on data and analysis transparency, QIIME 2 enables researchers to start an analysis with raw DNA sequence data and finish with publication-quality figures and statistical results.
+
+Visit [https://qiime2.org](https://qiime2.org) to learn more about the QIIME 2 project.
+
+## Installation
+
+Detailed [instructions](https://docs.qiime2.org/2019.4/install/) are available in the documentation.
```

Comment:
How about we drop the `/2019.4....` part, we will never remember to update that with each release.